mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
+
+DEBIAN=0
+
+if [ "$ID" == 'ubuntu' ] ; then
+    DEBIAN=1
+fi
+
+if [ $DEBIAN = 1 ] ; then
+    dpkg-buildpackage -us -uc
+else
+    tito build --rpm
+fi
+
+if [ -n "$BRANCH" ]; then
+    # A user explicitly told Jenkins to build a branch.
+    # (ie, this is not an automatic GitHub pull request build.)
+    # Move the artifacts into position for archiving.
+    cd $WORKSPACE
+    rm -rf $BRANCH
+    mkdir $BRANCH
+
+    if [ $DEBIAN = 1 ] ; then
```

Comment:
@alfredodeza your point is valid about the "`DEBIAN`" variable only being set on Ubuntu. If we really want to clean that up, I propose that we first do it to the takora job, so that this one matches.